zipkin-ui/static/trace_it_IT.properties

```diff
@@ -0,0 +1,7 @@
+trace.duration = Durata:
+trace.service = Servizi:
+trace.depth = Profinditá:
+trace.spans = Spans totali:
```

Comment:
nit: you usually don't add the plural "s" when using English words in Italian. I'd say "Span totali:"